packager/mpd/base/mpd_utils.cc

```diff
@@ -354,6 +359,23 @@ Element GenerateCencPsshElement(const std::string& pssh) {
   return cenc_pssh;
 }
 
+// Extract MS PlayReady Object from given PSSH
+// and encode it in base64
+Element GenerateMsprProElement(const std::string& pssh)
+{
```

Comment:
Can you move it to line 364 to be consistent to other functions?

Reply:
Sure